specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.1-preview.3/FormRecognizer.json

```diff
@@ -0,0 +1,2357 @@
+      "enum": [
+        "string",
+        "date",
+        "time",
+        "phoneNumber",
+        "number",
+        "integer",
+        "array",
+        "object",
+        "selectionMark"
+      ],
+      "x-ms-enum": {
+        "name": "FieldValueType",
+        "modelAsString": false
+      },
+      "x-nullable": false
+    },
+    "Confidence": {
+      "description": "Confidence value.",
+      "type": "number",
+      "minimum": 0,
+      "maximum": 1,
+      "x-nullable": false
+    },
+    "Appearance": {
+      "description": "An object representing the appearance of the text line.",
+      "type": "object",
+      "required": [
+        "style",
+        "styleConfidence"
+      ],
+      "properties": {
+        "style": {
+          "description": "Style of the extracted text: handwriting or printed.",
+          "enum": [
+            "handwriting",
+            "printed"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "textStyle",
+            "modelAsString": true
+          },
+          "x-nullable": false
+        },
+        "styleConfidence": {
+          "description": "Confidence value of predicted style.",
+          "$ref": "#/definitions/Confidence"
+        }
+      }
+    }
+  },
+  "parameters": {
+    "Endpoint": {
+      "name": "endpoint",
+      "description": "Supported Cognitive Services endpoints (protocol and hostname, for example: https://westus2.api.cognitive.microsoft.com).",
+      "x-ms-parameter-location": "client",
+      "required": true,
+      "type": "string",
+      "in": "path",
+      "x-ms-skip-url-encoding": true
+    },
+    "FileStream": {
+      "name": "fileStream",
+      "description": ".json, .pdf, .jpg, .png or .tiff type file stream.",
+      "x-ms-parameter-location": "method",
+      "in": "body",
+      "schema": {
+        "$ref": "#/definitions/SourcePath"
+      }
+    },
+    "PageRange": {
+      "name": "PageRange",
+      "in": "query",
+      "description": "Custom page numbers for multi-page documents(PDF/TIFF), input the number of the pages you want to get OCR result. For a range of pages, use a hyphen. Separate each page or range with a comma or space.",
+      "required": false,
+      "x-ms-parameter-location": "method",
+      "type": "array",
+      "items": {
+        "type": "string",
+        "pattern": "(^[0-9]+-[0-9]+$)|(^[0-9]+$)"
+      }
+    },
+    "Locale": {
+      "name": "locale",
+      "in": "query",
+      "description": "Locale of the input document. Supported locales include: en-AU, en-CA, en-GB, en-IN, en-US(default).",
+      "required": false,
+      "x-ms-parameter-location": "method",
+      "type": "string",
+      "enum": [
+        "en-AU",
+        "en-CA",
+        "en-GB",
+        "en-IN",
+        "en-US"
+      ],
+      "x-ms-enum": {
+        "name": "Locale",
+        "modelAsString": true
+      }
+    },
+    "Language": {
```

Comment:
This is the Language parameter definition we should be updating.

Reply:
done.